values.yaml

```diff
@@ -1,402 +1,134 @@
-# storageClassName -- Sets the storage class for all Coder services and user
-# environments. By default, the storageClassName is unspecified, and thus the
-# default StorageClass is used. If storageClassName is not specified and a
-# default StorageClass does not exist, then the deployment will fail. The
-# storageClass MUST support the ReadWriteOnce access mode.
-storageClassName: ""
-
-# serviceType -- See the following for the different serviceType options and
-# their use:
-# https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
-serviceType: "ClusterIP"
-
-# podSecurityPolicyName -- The name of the pod security policy to apply to all
-# Coder services and user environments. The optional ingress has its own field
-# for pod security policy as well.
-podSecurityPolicyName: ""
-
-# clusterDomainSuffix -- If you've set a custom default domain for your
-# cluster, you may need to remove or change this DNS suffix for service
-# resolution to work correctly.
-clusterDomainSuffix: ".svc.cluster.local"
-
-# contains configuration for the bundled ingress controller.
-ingress:
-  # ingress.enable -- If set to true, a Coder compatible ingress kind will be
-  # created. You can configure it with `ingress.annotations` below.
-  enable: true
-  # ingress.useDefault -- If set to true, will deploy an nginx ingress that will
-  # allow you to access Coder from an external IP address, but only if your
-  # kubernetes cluster is configured to provision external IP addresses. If you
-  # would like to bring your own ingress and hook Coder into that instead, set
-  # this value to false.
-  useDefault: true
-  # ingress.usePathWildcards -- Whether or not the ingress object should use
-  # path wildcards, i.e., ending with "/*". Some ingresses require this
-  # while others do not. You should check which path style your ingress
-  # requires. For ingress-nginx this should be set to false.
-  usePathWildcards: false
-  # ingress.host -- The hostname to use for accessing the platform. This can
-  # be left blank, and the user can still access the platform from the external
-  # IP or a DNS name that resolves to the external IP address.
-  host: ""
```

Comment:
How does someone configure `ingress.host` for the first time - don't we have a chicken-and-egg situation since we need a host to set up TLS, and we need TLS in order to log in to the admin console securely?

Reply:
Is it that unreasonable for people to log into the admin panel over HTTP during setup? If they really don't want to they can setup their own certificates and set the secret names on coderd.tls

Reply:
Yeah, that's a fair point @deansheather, I agree with that